jersey/tests/connector/pom.xml

```diff
@@ -53,6 +53,11 @@
             <artifactId>helidon-common-testing-http-junit5</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
The connector requires http2 in its module-info.java, so the library must be a dependency - e.g. it should not be required here

Reply:
Roger